frameworks/digital-outcomes-and-specialists/questions/brief-responses/niceToHaveRequirements.yml

```diff
@@ -1,20 +1,27 @@
-name: Nice-to-have requirements
-question: Do you have any of the nice-to-have skills and experience?
+name: Do you have any of the nice-to-have skills or experience?
+question: Do you have any of the nice-to-have skills or experience?
 question_advice: |
-  Buyers will filter supplier applications based on the number of nice-to-have skills and experience they have.
+  All nice-to-have skills and experience are optional.
 
-type: boolean_list
-number_of_items: 20
-optional: true
+  The buyer will assess and score your evidence to shortlist the best suppliers.
 
-depends:
-  - "on": "lot"
-    being:
-      - digital-outcomes
-      - digital-specialists
-      - user-research-participants
-validations:
-  -
-    name: answer_required
-    message: 'You need to answer all the nice-to-have requirement questions.'
-empty_message: Respond to nice-to-have requirements
+  <h3>Evidence structure</h3>
```

Comment:
So you can put html into markdown (http://stackoverflow.com/questions/975264/how-do-i-set-an-html-class-attribute-in-markdown) but given it will give us a bit of a nasty mix I'd be tempted to stick with one language. (We also ran into problems before with markdown not rendering on the page as we wished if I remember correctly...)